Sources/XcbeautifyLib/Renderers/Microsoft/MicrosoftOutputRendering.swift

```diff
@@ -193,127 +187,66 @@ struct GitHubActionsRenderer: OutputRendering {
             message: warningMessage
         )
     }
 
-    func formatSwiftTestingRunCompletion(group: SwiftTestingRunCompletionCaptureGroup) -> String {
-        let outputString = "Test run with \(group.numberOfTests) tests passed after \(group.totalTime) seconds"
-        return outputGitHubActionsLog(annotationType: .notice, message: outputString)
-    }
-
     func formatSwiftTestingRunFailed(group: SwiftTestingRunFailedCaptureGroup) -> String {
         let errorMessage = "Test run with \(group.numberOfTests) tests failed after \(group.totalTime) seconds with \(group.numberOfIssues) issue(s)"
-        return outputGitHubActionsLog(
-            annotationType: .error,
+        return makeOutputLog(
+            annotation: .error,
             message: errorMessage
         )
     }
 
     func formatSwiftTestingSuiteFailed(group: SwiftTestingSuiteFailedCaptureGroup) -> String {
         let errorMessage = "Suite \(group.suiteName) failed after \(group.timeTaken) seconds with \(group.numberOfIssues) issue(s)"
-        return outputGitHubActionsLog(
-            annotationType: .error,
+        return makeOutputLog(
+            annotation: .error,
             message: errorMessage
         )
     }
 
     func formatSwiftTestingTestFailed(group: SwiftTestingTestFailedCaptureGroup) -> String {
         let errorMessage = "\(group.testName) (\(group.timeTaken) seconds) \(group.numberOfIssues) issue(s)"
-        return outputGitHubActionsLog(
-            annotationType: .error,
+        return makeOutputLog(
+            annotation: .error,
             message: errorMessage
         )
     }
 
-    func formatSwiftTestingTestSkipped(group: SwiftTestingTestSkippedCaptureGroup) -> String {
-        let message = "Skipped \(group.testName)"
-        return outputGitHubActionsLog(
-            annotationType: .notice,
-            message: message
-        )
-    }
-
-    func formatSwiftTestingTestSkippedReason(group: SwiftTestingTestSkippedReasonCaptureGroup) -> String {
-        let message = "Skipped \(group.testName)" + (group.reason.map { ".(\($0))" } ?? "")
-        return outputGitHubActionsLog(
-            annotationType: .notice,
-            message: message
-        )
-    }
-
     func formatSwiftTestingIssue(group: SwiftTestingIssueCaptureGroup) -> String {
         let message = "Recorded an issue" + (group.issueDetails.map { " (\($0))" } ?? "")
-        return outputGitHubActionsLog(
-            annotationType: .notice,
+        return makeOutputLog(
+            annotation: .error,
             message: message
         )
     }
 
     func formatSwiftTestingIssueArguments(group: SwiftTestingIssueArgumentCaptureGroup) -> String {
         let message = "Recorded an issue" + (group.numberOfArguments.map { " (\($0)) argument(s)" } ?? "")
-        return outputGitHubActionsLog(
-            annotationType: .notice,
+        return makeOutputLog(
+            annotation: .error,
             message: message
         )
     }
```

Comment:
I changed the annotation for these two methods to `.error` as it reflects what's shown in the Xcode.

Reply:
Thanks! If it's not too much trouble, can you split this out into its own PR? Happy to do that if you'd like – just let me know if you'd like me to.

Reply:
Sure, I will raise a separate PR soon.